codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/util/Node.kt

```diff
@@ -0,0 +1,31 @@
+package software.amazon.smithy.kotlin.codegen.rendering.util
+
+import software.amazon.smithy.kotlin.codegen.utils.dq
+import software.amazon.smithy.model.node.ArrayNode
+import software.amazon.smithy.model.node.BooleanNode
+import software.amazon.smithy.model.node.Node
+import software.amazon.smithy.model.node.NullNode
+import software.amazon.smithy.model.node.NumberNode
+import software.amazon.smithy.model.node.ObjectNode
+import software.amazon.smithy.model.node.StringNode
+
+/**
+ * Renders a [Node] into String format for codegen.
+ */
+fun Node.render(): String = when (this) {
```

Comment:
naming: render usually means you are making a `write(...)` call, which does not happen here